PixelHunter1995/Screen.cs

```diff
@@ -92,24 +102,12 @@ private Rectangle RenderTargetFullScreenRect()
                     fullScreenWidth, newWindowHeight);
             }
         }
-
-        private bool IsAltEnterPressed(KeyboardState state)
-        {
-            return (state.IsKeyDown(Keys.LeftAlt) || state.IsKeyDown(Keys.RightAlt)) && state.IsKeyDown(Keys.Enter);
-        }
-
-        public void CheckForFullScreen()
+        
+        public void CheckForFullScreen(Input input)
         {
-            KeyboardState state = Keyboard.GetState();
-            if (!justToggledFullscreen && IsAltEnterPressed(state))
+            if (input.Actions.GetState(Action.ToggleFullscreen).IsEdgeDown)
```

Comment:
The sequence of calls here is quite confusing. I feel like we should get rid of at least one of them.

Reply:
I agree. I will remove the CheckForFullscreen method (why were a method named "check" actually changing something too?), and instead have it such that Main.Update checks the ToggleFullscreen action and calls Screen.ToggleFullscreen.